grader/reject_test.go

```diff
@@ -1,14 +1,15 @@
 package grader
 
 import (
-    "path/filepath"
-    "reflect"
-    "testing"
-
-    "github.com/edulinq/autograder/common"
-    "github.com/edulinq/autograder/config"
-    "github.com/edulinq/autograder/db"
-    "github.com/edulinq/autograder/model"
+	"path/filepath"
```

Comment:
Inconsistent use of spaces and tabs in this file.

We have been meaning to make a ruling on go fmt, but until then try to keep files consistent.

Reply:
Fixed in 86203ab75f905f891ae7c49699ebcb5fbfe8a0ad

I must have run `go fmt` by accident. It would be great if we could adapt the formatter across the repository :+1: